src/gui/search/pluginselectdlg.h

```diff
@@ -34,11 +34,11 @@
 
 #include <QDialog>
 
+#include "base/searchengine.h"
```

Comment:
nope, another way will require duplicate typedef here as:
```c++
#include "base/utils/version.h"
using PluginVersion = Utils::Version<unsigned short, 2>;
```

the current way is much simpler.